src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
+| Multi-echo gradient echo                   | MEGRE   | Grouping | Anatomical gradient echo images acquired at different echo times. _Associated output suffixes_: T2starmap, R2starmap                                                                                                                                                                                                                                                                                                                                             |
+| Magnetization transfer ratio               | MTR     | Grouping | Anatomical images for calculating a semi-quantitative magnetization transfer ratio map. _Associated output suffixes_: MTRmap                                                                                                                                                                                                                                                                                                                                                                                 |
+| Magnetization transfer saturation          | MTS     | Grouping | Anatomical images for calculating a semi-quantitative magnetization transfer saturation index map. The MTS method involves three sets of anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff) and flip angle ([Helms et al. 2008](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.21732)). _Associated output suffixes_: T1map, MTsat                                                                                                                                                                                               |
+| Multi-parametric mapping                   | MPM     | Grouping | Anatomical images for multiparametric mapping (a.k.a hMRI). The MPM approaches involves the acquisition of highly-similar anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff), flip angle and (optionally) echo time and magnitue/phase parts ([Weiskopf et al. 2013](https://www.frontiersin.org/articles/10.3389/fnins.2013.00095/full)). See [here](https://owncloud.gwdg.de/index.php/s/iv2TOQwGy4FGDDZ) for suggested MPM acquisition protocols.,_Associated output suffixes_:R1map, R2starmap, MTsat, PDmap, T1map, T2starmap                                               |
+
+For example:
+
+```Text
+sub-01_fa-1_VFA.nii.gz
+sub-01_fa-1_VFA.json
+sub-01_fa-2_VFA.nii.gz
+sub-01_fa-2_VFA.json
+```
+
+Please see the [entity table appendix]() for the REQUIRED and OPTIONAL entities
+for each `grouping suffix`. 
```

Comment:
As I had mentioned in #371 in relation to this BEP, I think introducing new definitions such as "grouping suffix" IMHO should be suggested as an independent PR first and/or in parallel to the BEP.  That would allow for a better targeted review  of the proposed functionality.  Clumping everything together into a single PR makes it hard to impossible to provide adequate analysis/review of all aspects introduced and/or changed.

Also such new definitions should also be added to Glossary.
NB * - I do not think we have a term like that yet, but it is pretty much a list of "definitions" we have already in glossary: https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#definitions

Reply:
I think the "grouping suffix" could be a concept relevant beyond this BEP, and it should be, perhaps, added to the [Common principles](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#file-name-structure).